File: engine/model/src/main/scala/tools/packet/guild/GuildQuestWaitingNotice.scala
package tools.packet.guild

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class GuildQuestWaitingNotice(private var _channel: Byte, private var _position: Int) extends PacketInput {
  def channel: Byte = _channel

  def position: Int = _position

  override def opcode(): SendOpcode = SendOpcode.GUILD_OPERATION
}

File: engine/engine-model/src/main/java/tools/packet/guild/GuildQuestWaitingNotice.java
package tools.packet.guild;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record GuildQuestWaitingNotice(Byte channel, Integer position) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.GUILD_OPERATION;
   }
}